bundles/org.openhab.binding.freeathomesystem/README.md

```diff
@@ -0,0 +1,147 @@
+# ABB/Busch-free@home Smart Home binding
+
+openHAB ABB/Busch-free@home binding based on the offical free@home local API.
+
+# Description
+
+This binding allows you to connect your free@home Smart Home system from ABB / Busch-Jaeger to openHAB and to control and observe most of the components.
+It requires a System Access Point with version 2.6.1 or higher.
+
+# Supported Devices
+
+**Network Gateway / System Access Point**
+
+ - ABB / Busch-Jaeger System Access Point 2.0
+
+**Sensors and Actuators**
+
+ - Switch Actuator Sensor with single and multiple channels (wired and wireless)
+ - Dimming Actuator Sensor with single and multiple channels (wired, wireless and flex)
+ - Motion detection with and without actuator (wired, wireless and flex)
+ - Switch Actuator 4-channel
+ - Dimming Actuator 4- and 6-channel
+ - Door opener actuator
+ - Door ring sensor
+ - Hue devices (untested)
+
+**Blinds and Windows**
+
+ - Shutter Actuator with single and multiple channels (wired and wireless)
+ - Blind Actuator  with single and multiple channels (wired and wireless)
+ - Attic window actuator
+ - Awning actuator
+
+**Room Temperature Control**
+
+ - Room temperature controller master without fan
+ - Room temperature controller master with fan
+ - Room temperature controller slave
+
+**Other devices** (e.g. movement detector, ring sensor and door opener)
+
+ - IP-touch panel (function: door opener, door ring sensor)
+ - Virtual devices (e.g. virtual switch, RTC and detectors)
+
+**Information about virtual devices**
+Virtual device in the free@home smart-home system needs continuous keep-alive signal otherwise the free@home device is marked as unresponsive.
+This keep-alive signal must be provided by a user script or set the TTL value of the virtual device to "-1" during the creation of the virtual device.
+
+# Tested SysAP Versions
+
+| Version | Supported |
+|---------|-----------|
+| 2.6.1   | yes       |
+| 2.6.3   | yes       |
+| 3.1.1   | yes       |
+
+# Setup / Installation
+
+## Prerequisites
+
+To make use of this Binding first the local free@home API has to be activated.
+The API is disabled by default.
+
+1. Open the free@home next app
+1. Browse to "Settings ⇨ free@home settings ⇨ local API and activate the checkbox
+
+## Setup and Discovery
+
+The free@home bridge shall be added manually.
```

Comment:
Out of curiosity, it does not support UPnP or mDNS discovery (or any other kind)? The app can't discover it?

Reply:
I am a big friend of automatic processes to improve the user experience. This is the reason that at the beginning my binding supported only the automatic device discovery and no manual config.

BUT the UPnP is unfortunately not supported in the free@home gateway. The Connection can be established only on the classical way, even the own free@home App is using the classical methods.

Reply:
Thanks for confirming. The reason for asking is that we now have an addon finder for suggesting bindings to install, and this would have been a great way to promote your new binding.